src/controllers/appointment.ts

```diff
@@ -24,12 +24,40 @@ const getAppointments = async (
       data,
       message: 'appointment successful',
     });
-  } catch (err:unknown) {
+  } catch (err: unknown) {

```

Comment:
```suggestion
  } catch (err:) {

```
u can remove unknown from catch block